src/generators.rs

```diff
@@ -409,12 +433,13 @@ pub fn path_graph(
 ///       os.remove(tmp_path)
 ///   image
 ///
-#[pyfunction(inward = "false")]
-#[text_signature = "(/, num_nodes=None, weights=None, inward=False)"]
+#[pyfunction(inward = "false", bidirectional = "false")]
+#[text_signature = "(/, num_nodes=None, weights=None, inward=False, bidirectional=False)"]
 pub fn directed_star_graph(
     py: Python,
     num_nodes: Option<usize>,
     weights: Option<Vec<PyObject>>,
+    bidirectional: bool,
     inward: bool,
```

Comment:
Could you flip these? I just think it'll make theupgrade path a bit simpler for any potential rust callers of this function.

```suggestion
    inward: bool,
    bidirectional: bool,
```